php/composer.json

```diff
@@ -1,9 +1,16 @@
 {
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/stanley-cheung/Protobuf-PHP"
+    }
+  ],
   "name": "grpc/grpc-demo",
   "description": "gRPC example for PHP",
   "minimum-stability": "dev",
   "require": {
     "php": ">=5.5.0",
+    "datto/protobuf-php": "dev-master",
```

Comment:
So what happen here is that, I am importing the official package `datto/protobuf-php` from Packagist, and replace it with my fork. Apparently that's the way to specify that in the `composer.json` file.